test/integration/options.spec.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var path = require('path');
+var loadOptions = require('../../lib/cli/options').loadOptions;
+
+describe('options', function() {
+  it('Should support extended options', function() {
```

Comment:
[Style] Nit: other tests in this repo generally lower-case the 's'. For consistency...

```suggestion
  it('should support extended options', function() {
```